test/unittests/api_test.cpp

```diff
@@ -217,6 +217,40 @@ TEST(api, resolve_imported_function_duplicate)
     EXPECT_THAT(execute(*instance, 1, {0_u32}), Result(42));
 }
 
+TEST(api, resolve_imported_function_duplicate_with_context)
```

Comment:
The point of this test to check that even though the `any` is duplicated in each ExternalFunction, they are somehow still reference to the same?